doc/content/reference/api/field-mask.md

```diff
@@ -6,17 +6,185 @@ weight: -1
 
 ## Field Masks
 
-{{% tts %}} APIs use field masks to specify a subset of fields that should be returned by a reading request, or to specify fields that should be updated in a writing request. See Google's [Protocol Buffers reference](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.FieldMask) for more information about field masks.
+{{% tts %}} APIs use field masks to specify a subset of fields that should be returned by a reading request, or to specify fields that should be updated in a writing request. By default, API requests will not return or update most fields, unless they are specified in a field mask.
+
+The following fields are always returned and always updated:
```

Comment:
These fields are always returned for messages that have them. 
Only `updated_at` is automatically updated.